### PR TITLE
SCW-346: Entry Point Versioning

### PR DESCRIPTION
**Pull request by maxisch**
## Description

Call crossbit main which returns the appropiate entry point version.

## Test plan

Manual testing

### CODE REVIEW

File: packages/client/wallets/aa/package.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
     },
     "devDependencies": {
         "@playwright/test": "^1.41.2",
-        "@types/node": "^20.11.20"
+        "@types/node": "^20.11.20",
+        "tsup": "^8.0.2"
```
Comment (MatiNavCrossmint) [inline packages/client/wallets/aa/package.json:35]: Why is this added?

File: packages/client/wallets/aa/src/CrossmintAASDK.ts [hunk 2]
```diff
@@ -142,4 +146,22 @@ export class CrossmintAASDK {
             localStorage.removeItem(key);
         });
     }
+
+    private getEntryPointAddress(entryPointVersion: EntryPointVersion): EntryPoint {
+        return entryPointVersion === 0.6 ? ENTRYPOINT_ADDRESS_V06 : ENTRYPOINT_ADDRESS_V07;
+    }
```
Comment (MatiNavCrossmint) [inline packages/client/wallets/aa/src/CrossmintAASDK.ts:152]: As this is one line, I don't see the value of splitting this in two functions, which implies having to call both instead of one

**Review by MatiNavCrossmint — COMMENTED** [review]
Cool, just left some minus comments!

File: packages/client/wallets/aa/package.json (hunk 1 above)
Comment (maxisch) [reply packages/client/wallets/aa/package.json:35]: To enable pnpm dev appart from yarn dev

File: packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts [hunk 3]
```diff
@@ -50,39 +49,42 @@ export class EVMAAWallet<B extends EVMBlockchainIncludingTestnet = EVMBlockchain
...
     chain: B;
 
     constructor(
         account: KernelSmartAccount<EntryPoint, HttpTransport, TChain>,
         crossmintService: CrossmintWalletService,
         chain: B,
         publicClient: PublicClient,
-        ecdsaValidator: KernelValidator<entryPoint, "ECDSAValidator">
+        ecdsaValidator: KernelValidator<entryPoint, "ECDSAValidator">,
+        entryPoint: EntryPoint
```
Comment (afeight) [inline packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts:61]: just fyi you can shortcut and remove line 52 by doing:
```suggestion
        private entryPoint: EntryPoint
```

**Review by afeight — APPROVED** [review]
looks good to me
